pkg/proxy/iptables/proxier.go

```diff
@@ -1571,7 +1571,8 @@ func (proxier *Proxier) syncProxyRules() {
 	glog.V(5).Infof("Restoring iptables rules: %s", proxier.iptablesData.Bytes())
 	err = proxier.iptables.RestoreAll(proxier.iptablesData.Bytes(), utiliptables.NoFlushTables, utiliptables.RestoreCounters)
 	if err != nil {
-		glog.Errorf("Failed to execute iptables-restore: %v\nRules:\n%s", err, proxier.iptablesData.Bytes())
+		glog.Errorf("Failed to execute iptables-restore: %v", err)
+		glog.V(2).Infof("Rules:\n%s", proxier.iptablesData.Bytes())
```

Comment:
This is an error... it shouldn't be happening at all. Why is it happening so much that it causes problems?